flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_RequestsWeather.js

```diff
@@ -91,9 +91,13 @@ class CDUAocRequestsWeather {
         };
 
         mcdu.onRightInput[5] = async () => {
+            const icaos = [store["arpt1"], store["arpt2"], store["arpt3"], store["arpt4"]];
+            if (icaos[0] == "" && icaos[1] == "" && icaos[2] == "" && icaos[3] == "") {
```

Comment:
**question**: Why do you add these to an array?
**question**: Do you do non-strict comparison (`==` instead of `===`) on purpose?

Reply:
Nope, that was my inner Java-thinking coming through xD...
Impl. `Array.every`. Simply didn't know about that function  :/